examples/components/utils/aggregatemodelweights/spec.yaml

```diff
@@ -44,7 +52,8 @@ command: >-
   $[[${{inputs.input_silo_3}}]] 
   $[[${{inputs.input_silo_4}}]] 
   $[[${{inputs.input_silo_5}}]] 
-
+  --ancillary_files $[[${{inputs.ancillary_files}}]] 
+  --out_checkpoint_name $[[${{inputs.out_checkpoint_name}}]] 
```

Comment:
same here, the `--out_checkpoint_name` has to be within the optional block `$[[..]]`